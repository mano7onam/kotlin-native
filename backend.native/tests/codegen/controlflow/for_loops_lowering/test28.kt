package codegen.controlflow.for_loops_lowering.test28

import kotlin.test.*

@Test fun runTest() {
	val arr = arrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
	val arr1 = arrayOf("a", "hello world", "aaa")
	for (i in 0 .. 10 step 2) {
		for (j in arr) {
			for (k in arrayOf(10.0)) {
				for (t in arr1) {
					println(j)
					println(k)
					println(t)
				}			
			}	    	
	    }
	}   
}
