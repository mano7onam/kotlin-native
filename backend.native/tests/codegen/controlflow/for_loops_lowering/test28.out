1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
1
10.0
a
1
10.0
hello world
1
10.0
aaa
2
10.0
a
2
10.0
hello world
2
10.0
aaa
3
10.0
a
3
10.0
hello world
3
10.0
aaa
4
10.0
a
4
10.0
hello world
4
10.0
aaa
5
10.0
a
5
10.0
hello world
5
10.0
aaa
6
10.0
a
6
10.0
hello world
6
10.0
aaa
7
10.0
a
7
10.0
hello world
7
10.0
aaa
8
10.0
a
8
10.0
hello world
8
10.0
aaa
9
10.0
a
9
10.0
hello world
9
10.0
aaa
10
10.0
a
10
10.0
hello world
10
10.0
aaa
